plugins/systemd/timesync_status

```diff
@@ -53,6 +53,9 @@ def parse_time(value):
     if ' ' in value:
         return sum(parse_time(x) for x in value.split(' '))
 
+    if value == "0":
```

Comment:
Under which circumstances would this value appear?

Would it be more appropriate to return `None` or raise an exception in this case?

Reply:
If time is exactly zero (for example Jitter), there is no unit (suffix) and re.match will fail. If systemd says it's zero, it means zero, not errror. :)

Reply:
example:
```
$ timedatectl timesync-status
       Server: #REDACTED#
Poll interval: 1min 4s (min: 32s; max 34min 8s) 
         Leap: normal
      Version: 4
      Stratum: 2
    Reference: C37190EE
    Precision: 62us (-14)
Root distance: 67.641ms (max: 5s) 
       Offset: +191us
        Delay: 537us
       Jitter: 0
 Packet count: 1
    Frequency: +3.394ppm